duckduckgo/Android
# Refresh versions

Task/Issue URL: https://app.asana.com/0/0/1201407021141151/f

### Description

Upgraded a number of libraries. Each commit contains changes for one library. Tested the app.

### Steps to test this PR

1. Install from this branch 
2. Use the app and should not see an issue.
3. What was specifically tested related to upgraded libraries:
        - all fire animations
        - exporting and importing bookmarks 
        - screen rotation & app in background / foreground
        - various screens in the app
        - ran all tests

### No UI changes
